<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AllSafeDumper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabSites">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_import"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_addSite"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_HideShowDelitedSite"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_import">
   <property name="text">
    <string>Импортировать</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Экспортировать</string>
   </property>
  </action>
  <action name="action_addSite">
   <property name="text">
    <string>Добавить сайт</string>
   </property>
  </action>
  <action name="action_HideShowDelitedSite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вкл/Выкл удаления сайтов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
